src/pages/saksbehandling/søknadsbehandling/opphold-i-utlandet/OppholdIUtlandet.tsx

```diff
@@ -62,130 +81,111 @@ const OppholdIUtlandet = (props: VilkårsvurderingBaseProps) => {
-                        >
-                            <Radio
-                                label={intl.formatMessage({ id: 'radio.label.ja' })}
-                                name="status"
-                                onChange={() =>
-                                    formik.setValues((v) => ({
-                                        ...v,
-                                        status: OppholdIUtlandetStatus.SkalVæreMerEnn90DagerIUtlandet,
-                                    }))
-                                }
-                                checked={formik.values.status === OppholdIUtlandetStatus.SkalVæreMerEnn90DagerIUtlandet}
-                            />
-                            <Radio
-                                label={intl.formatMessage({ id: 'radio.label.nei' })}
-                                name="status"
-                                onChange={() =>
-                                    formik.setValues((v) => ({
-                                        ...v,
-                                        status: OppholdIUtlandetStatus.SkalHoldeSegINorge,
-                                    }))
-                                }
-                                checked={formik.values.status === OppholdIUtlandetStatus.SkalHoldeSegINorge}
-                            />
-                            <Radio
-                                label={intl.formatMessage({ id: 'radio.label.uavklart' })}
-                                name="status"
-                                onChange={() =>
-                                    formik.setValues((v) => ({
-                                        ...v,
-                                        status: OppholdIUtlandetStatus.Uavklart,
-                                    }))
-                                }
-                                checked={formik.values.status === OppholdIUtlandetStatus.Uavklart}
-                            />
-                        </RadioGruppe>
+                        <Controller
+                            control={form.control}
+                            name="status"
+                            render={({ field, fieldState }) => (
+                                <RadioGruppe
+                                    legend={formatMessage('radio.oppholdIUtland.legend')}
+                                    feil={fieldState.error?.message}
+                                    onBlur={field.onBlur}
+                                >
+                                    <Radio
+                                        id={field.name}
+                                        radioRef={field.ref}
+                                        label={formatMessage('radio.label.ja')}
+                                        name="status"
+                                        onChange={() =>
+                                            field.onChange(OppholdIUtlandetStatus.SkalVæreMerEnn90DagerIUtlandet)
```

Comment:
Denne enumen føles litt inkonsekvent med de tidligere, som har brukt Oppfylt/IkkeOppfylt. Vet ikke hva som er best.

Reply:
Heh, ja. Er nok gjort sånn for at det skal være enklere å relatere det direkte til det man ser i GUI. Følte uansett at det var litt utenfor skop i denne PR-en å begynne å grave så dypt i sånne detaljer.